form_components/ProviderForm.js

```diff
@@ -75,13 +76,21 @@ const styles = StyleSheet.create({
         marginBottom: 5,
         fontSize: 18,
         borderRadius: 6,
-        backgroundColor: '#EDEFEF'
+        backgroundColor: '#EDEFEF',
```

Comment:
Same comment about GlobalColors.

```suggestion
        backgroundColor: GlobalColors.backgroundColor,
```